R/plumber.R

```diff
@@ -493,7 +493,13 @@ plumber <- R6Class(
 
 #' @rdname plumber
 #' @export
-plumb <- function(file){
+plumb <- function(file, dir){
+  if(!xor(missing(file), missing(dir))){
+    stop("plumber needs only one of a file or a directory with a `plumber.R` file in its root.")
+  } else if (missing(file)){
+    stri_trim_right(dir, pattern = "[/]")
+    file <- stri_join(dir,"plumber.R", sep="/")
```

Comment:
Same `stri` concern, and I'm not sure if this will work on Windows. Let's use `file.path` instead.